requirements/edx/base.txt

```diff
@@ -33,7 +33,7 @@ django-method-override==0.1.0
 git+https://github.com/edx/django-rest-framework.git@3c72cb5ee5baebc4328947371195eae2077197b0#egg=djangorestframework==3.2.3
 django==1.8.7
 edx-rest-api-client==1.2.1
-elasticsearch==0.4.5
+edx-search==0.1.1
```

Comment:
The latest version of edx-search hosted on [PyPI](https://pypi.python.org/pypi/edx-search/1.0.0) is 1.0.0. I'm guessing you bumped the version so that Travis would push the new tag to PyPI? Regardless, I'd use 1.0.0 instead of 0.1.1, seeing as how it's the latest version.